client/jira.go

```diff
@@ -72,6 +72,14 @@ type (
 		Issues       []domain.Issue `json:"issues"'`
 	}
 
+	TransitionIdRequest struct {
+		TransitionId string `json:"id"`
+	}
+
+	TransitionRequest struct {
+		Transition TransitionIdRequest `json:"transition"`
+	}
+
 	AssignRequest struct {
```

Comment:
Change to `Assignee`

Reply:
@pamelin This part isn't related to the PR (the same as the "create" command). If you want, you can open a ticket for renaming/refactoring things. Thank you!!

Reply:
Seriously do we need a ticket for this?

Reply:
Done - I've fixed the name. Sorry, maybe I did miscommunicate what I meant. I mean if we want to do refactoring for all structs, variables, etc.. why don't we create a ticket for that? Thank you!